### ISSUE TITLE
Rule: equals sign should be on same line

### ISSUE THREAD

**Issue by MyDogTom** [shyiko/ktlint#105]
I would like to enforce this rule

>When a property initializer does not fit on a single line, break after the equals sign (=) and use a continuation indent.

[Source](https://android.github.io/kotlin-guides/style.html#properties)

Valid example
```kotlin
private val defaultCharset: Charset? =
        EncodingRegistry.getInstance().getDefaultCharsetForPropertiesFiles(file)
```

Violation example
```kotlin
private val defaultCharset: Charset?
        = EncodingRegistry.getInstance().getDefaultCharsetForPropertiesFiles(file)
```

I also would like to use same rule with functions.

Valid example
```kotlin
fun sum(a: Int, b: Int) = 
        a + b
```

Violation example
```kotlin
fun sum(a: Int, b: Int) 
        = a + b
```

PS. I'm happy to try implement it, if such PR is welcome.

**Comment by shyiko** [comment]
Hi @MyDogTom. Absolutely. I'm kinda surprised it's not enforced yet so PR is definitely more than welcome.
